ospf6d/ospf6_message.c

```diff
@@ -1937,6 +1937,27 @@ static int ospf6_write(struct thread *thread)
 				   oi->interface->name);
 			zlog_debug("    src: %s", srcname);
 			zlog_debug("    dst: %s", dstname);
+			switch (oh->type) {
```

Comment:
Inside all _print functions specific to ospf packet types there is another check for SEND and RECV. Can we remove that, it's just an overhead. I feel there is no need to segregate between SEND_HDR and SEND in this flow.

Reply:
I'm not sure I understand this. The _HDR flags were introduced to reduce the logging in cases where the logging can cause neighbor failures by making the processes run too long. The SEND check is much less expensive than the log it protects.